TODO.md

```diff
@@ -1,10 +1,23 @@
 # TODOs
 
-1. &#10003; Define the LIonCore M3 as TypeScript types.
-2. &#10003; Define the same thing meta-circularly, using those types.
-3. &#10003; Implement a syntax checker, specifically: no `unresolved`s in (at most) single-valued references.
+* &#10003; Define the LIonCore/M3 as TypeScript types.
```

Comment:
How TODOs compare to issues?
I think that having just issues (possibly with labels such as "Refinement", "Feature", etc.) could help and encourage participation

Reply:
At the moment, this is a list of things I expect to do pretty soon - possibly in 2022 still :)
But you're right: these should be issues. I'll make a meta-issue for that in a minute.